lib/JSON/Validator/OpenAPI/Dancer2.pm

```diff
@@ -0,0 +1,72 @@
+package JSON::Validator::OpenAPI::Dancer2;
+
+use Carp qw(confess);
+use Hash::MultiValue;
+use Mojo::Base 'JSON::Validator::OpenAPI';
+
+### request ###
+
+sub _get_request_data {
+  my ($self, $dsl, $in) = @_;
```

Comment:
That is an instance of [`Dancer2::Core::DSL`](https://metacpan.org/pod/Dancer2::Core::DSL) with the current request, etc. set in the background which provides all keywords from Dancer's *domain-specific language* (DSL). In this case it has the same purpose as `$c` in Mojolicious or Catalyst.